server/model/work.js

```diff
@@ -0,0 +1,14 @@
+// @flow
+
+export type Work = {|
+  id: string;
+  label: string;
+  descripition: string;
```

Comment:
From the API response:
![screen shot 2017-06-14 at 14 34 13](https://user-images.githubusercontent.com/1394592/27134799-9e0ae1f0-510e-11e7-80a7-df0b8d2f6ed3.png)